docs/source/ruleset.rst

```diff
@@ -0,0 +1,70 @@
+=======
+Ruleset
+=======
+
+This chapter explains how to use and take care of the ruleset. We assume you
+have peekaboo up and running and want to tweak or understand the default
+ruleset.
+
+We also asume you are familiar with python config parser.
+
+Section: rules
+==============
+Here rules can be disables by putting a ``#`` (comment) in front. Also the 
+order in which the rules will be processed can be changed by changing how
+the rules are listed (note that the trailing number is not relevant).
+
+
+Following sections
+==================
+The following sections are processed (if enabled in rules section) and
+contain for example the whitelist mime types. Comments also here can disables
+entries.
```

Comment:
Comments here can also disable entries?

Reply:
Individual entries within for example the whitelist can be disabled by putting an `#` in front.

Reply:
Better still!